### PR TITLE
add 0x setting

### PR DESCRIPTION
**Pull request by secmask**

### CODE REVIEW

**Comment by secmask** [comment]
@ductm54 I added `enabled` field.

File: reservesetting/storage/postgres/assets.go [hunk 1]
```diff
@@ -1041,6 +1074,15 @@ func (s *Storage) updateAsset(tx *sqlx.Tx, id rtypes.AssetID, uo storage.UpdateA
 		SanityRateProvider:    sanityRateProvider,
 		SanityRatePath:        sanityRatePath,
 	}
+	if uo.ZeroX != nil {
+		arg.updateZeroXPref = updateZeroXPref{
```
Comment (NNagato) [inline reservesetting/storage/postgres/assets.go:1078]: please check and add this param to `updateMsgs`, cuz, if `updateMsgs` is empty it's, will return an error in line 1194

**Review by NNagato — CHANGES_REQUESTED** [review]

File: reservesetting/storage/postgres/assets.go (hunk 1 above)
Comment (secmask) [reply reservesetting/storage/postgres/assets.go:1079]: nice catch, I fixed it.

**Review by NNagato — APPROVED** [review]
LGTM

**Pull request merged**